starknet/src/utils/into.cairo

```diff
@@ -0,0 +1,35 @@
+use array::{ArrayTrait, SpanTrait};
+use option::OptionTrait;
+use traits::Into;
+use integer::u256_from_felt252;
+use starknet::{ContractAddress, EthAddress};
+
+impl Felt252SpanIntoU256Array of Into<Span<felt252>, Array<u256>> {
```

Comment:
I think we can make it generic ?
```rust
impl<T, impl TCopy: Copy<T>, impl TIntoU: Into<T, U>> TSpanIntoU of Into<Span<T>, Array<U>>
  fn into(self: Span<T>) -> Array<U> {
   // ...
  }
```

might need some additional constraints on `U` since it's used by `ArrayTrait`.
If it's too difficult, we can drop the generics on U and just `impl<T> TSpanIntoU256`

Reply:
this seems excessive considering we only have 1 impl that would use it